### PR TITLE
en-NL-js

### PR DESCRIPTION
**Pull request by JeroenKleen**
#### Is this change backwards compatible or is it a breaking change?

description made on Oct.13 on request of BryanJ

### CODE REVIEW

File: src/js/messages/en-US.js [hunk 1]
```diff
@@ -1,125 +0,0 @@
-// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
```
Comment (alansouzati) [inline src/js/messages/en-US.js]: I assuming this deletion was an accident?

File: src/js/messages/en-NL.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import iconMessages from './icons/en-US';
+
+export default {
+  IndexFilters: {
+    filters: '{quantity, plural,\n  =0 {Filters}\n  =1 {one filter}\n  ' +
```
Comment (alansouzati) [inline src/js/messages/en-NL.js]: shouldn't we translate this as well?

File: src/js/messages/en-NL.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  Distribution: 'Distributie',
+  Email: 'E-mail',
+  'Enter Select': 'Druk op Enter om te selecteren',
+  Error: 'Foutmelding',
+  Filter: 'Filter',
+  Footer: 'voettekst',
+  Grommet: 'Grommet',
+  HotSpotsLabel: 'HotSpots: druk op de pijltjes toetsen voor bediening',
+  'GraphValues': (
+    'Graph has {count} items. Highest is {highest}' +
```
Comment (alansouzati) [inline src/js/messages/en-NL.js]: shouldn't we translate this as well?

File: src/js/messages/en-NL.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  'Previous Month': 'Vorige maand',
+  'Previous Slide': 'Vorige plaatje',
+  'Previous Tiles': 'Vorige Tegels',
+  // I'm using for Tiles the value that Microsoft uses in W10 as well what is called 'Tegels'
+  'Remember me': 'Onthouden',
+  Resource: 'Hulpmiddel',
+  Running: 'Lopende',
+  // Running could be as well be 'doorgaande' but its all depending on the usage where exactly
+  Search: 'Zoeken',
+  'Match Results': `There {count, plural,
```
Comment (alansouzati) [inline src/js/messages/en-NL.js]: shouldn't we translate this as well?

**Comment by alansouzati** [comment]
hey @JeroenKleen can you take a look in the comments in this PR ?

**Comment by tracybarmore** [comment]
@jeroenKleen, thanks for all your collaboration with us. We will be cleaning up our backlog first week of January 2017 by closing unresponsive issues that haven't been active for at least 30 days. 

If you would like this issue to remain open in our backlog, please comment to let us know. Thanks!

**Comment by tracybarmore** [comment]
Closing issue.

**Pull request closed without merging**